include/deal.II/base/polynomials_hermite.h

```diff
@@ -0,0 +1,157 @@
+// ---------------------------------------------------------------------
+//
+// Copyright (C) 2021 - 2022 by the deal.II authors
+//
+// This file is part of the deal.II library.
+//
+// The deal.II library is free software; you can use it, redistribute
+// it, and/or modify it under the terms of the GNU Lesser General
+// Public License as published by the Free Software Foundation; either
+// version 2.1 of the License, or (at your option) any later version.
+// The full text of the license can be found in the file LICENSE.md at
+// the top level directory of deal.II.
+//
+// ---------------------------------------------------------------------
+
+
+
+#ifndef dealii_hermite_polynomials
+#define dealii_hermite_polynomials
+
+#include <deal.II/base/config.h>
+
+#include <deal.II/base/exceptions.h>
+#include <deal.II/base/point.h>
+#include <deal.II/base/polynomial.h>
+#include <deal.II/base/subscriptor.h>
+
+#include <deal.II/lac/full_matrix.h>
+
+#include <algorithm>
+#include <exception>
+#include <memory>
+#include <vector>
+
+
+
+DEAL_II_NAMESPACE_OPEN
+
+/**
+ * @addtogroup Polynomials
+ * @{
+ */
+
+namespace Polynomials
+{
+  /**
+   * This class implements Hermite interpolation polynomials
```

Comment:
Could you add a reference?

Reply:
Yes, of course. Is there a standard format for adding academic references to deal.II documentation?

Reply:
E.g.: 

https://github.com/dealii/dealii/blob/e6779913e7f770652da442b3c2460dec3e864029/include/deal.II/base/mpi_consensus_algorithms.h#L316-L317

https://github.com/dealii/dealii/blob/8e3873780d4d35d18dfaf7c2a5364e39aa175402/doc/doxygen/references.bib#L1684-L1694